### PR TITLE
add cn vietnam

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: streams/vn_fptplay.m3u [hunk 1]
```diff
@@ -14,6 +14,9 @@ https://livecdn.fptplay.net/hda2/bbcearth_vhls.smil/chunklist.m3u8
 #EXTINF:-1 tvg-id="BoomerangSoutheastAsia.us" status="online" user-agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36",Boomerang South East Asia (Vietnamese dub) (1080p)
 #EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36
 https://livecdn.fptplay.net/hda3/boomerang_vhls.smil/chunklist.m3u8
+#EXTINF:-1 tvg-id="CartoonNetworkVietnam.vn" status="online" user-agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36",Cartoon Network Vietnam (Geo-blocked) (1080p)
```
Comment (UltraHDR) [inline streams/vn_fptplay.m3u:17]:
```suggestion
#EXTINF:-1 tvg-id="CartoonNetworkVietnam.vn" status="online" user-agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36",Cartoon Network Vietnam [Geo-blocked] (1080p)
```

**Comment by UltraHDR** [comment]
Link doesnt work as geo-blocked but approving anyway

**Review by UltraHDR — APPROVED** [review]

**Comment by LaneSh4d0w** [comment]
Well, every hda* links are geo-blocked. Links starting from hdb works everywhere.

**Review by BellezaEmporium — APPROVED** [review]

**Review by ghost — COMMENTED** [review]
bb

**Pull request merged**